gcs/electron/main.ts

```diff
@@ -150,6 +150,44 @@ function createLoadingWindow() {
   loadingWin.center()
 }
 
+function startBackend() {
+  if (pythonBackend) {
+    console.log('Backend already running');
+    return;
+  }
+
+  console.log('Starting backend');
+
+  // Add more platforms here
+  const backendPaths: Partial<Record<NodeJS.Platform, string>> ={
+    win32: 'extras/fgcs_backend.exe',
+    darwin: path.join(process.resourcesPath, '../extras', 'fgcs_backend.app', 'Contents', 'MacOS', 'fgcs_backend')
+  };
+
+  const backendPath = backendPaths[process.platform];
+
+  if (!backendPath) {
+    console.error('Unsupported platform!');
+    return;
+  }
+
+  console.log(`Starting backend: ${backendPath}`);
+  pythonBackend = spawn(backendPath);
+
+  // pythonBackend.stdout.on('data', (data) => console.log(`Backend stdout: ${data}`));
+  // pythonBackend.stderr.on('data', (data) => console.error(`Backend stderr: ${data}`));
```

Comment:
are these lines needed?

Reply:
Probably not. I just needed it to log out backend data while I test. It's not really necessary in production.

Reply:
We'll keep it just for future use i guess